src/integrationTest/java/uk/gov/hmcts/darts/authorisation/controller/AuthorisationControllerIntTest.java

```diff
@@ -0,0 +1,551 @@
+                                            1,
+                                            2
+                                          ],
+                                          "permissions": [
+                                            "VIEW_MY_TRANSCRIPTIONS",
+                                            "SEARCH_CASES",
+                                            "APPROVE_REJECT_TRANSCRIPTION_REQUEST",
+                                            "VIEW_MY_AUDIOS",
+                                            "EXPORT_PROCESSED_PLAYBACK_AUDIO",
+                                            "REQUEST_TRANSCRIPTION",
+                                            "REQUEST_AUDIO",
+                                            "VIEW_DARTS_INBOX",
+                                            "READ_TRANSCRIBED_DOCUMENT",
+                                            "LISTEN_TO_AUDIO_FOR_PLAYBACK",
+                                            "RETENTION_ADMINISTRATION"
+                                          ]
+                                        },
+                                        {
+                                          "roleId": 1,
+                                          "roleName": "JUDICIARY",
+                                          "globalAccess": true,
+                                          "courthouseIds": [
+                                            1,
+                                            2,
+                                            3
+                                          ],
+                                          "permissions": [
+                                            "VIEW_MY_TRANSCRIPTIONS",
+                                            "SEARCH_CASES",
+                                            "VIEW_MY_AUDIOS",
+                                            "READ_JUDGES_NOTES",
+                                            "EXPORT_PROCESSED_PLAYBACK_AUDIO",
+                                            "REQUEST_AUDIO",
+                                            "REQUEST_TRANSCRIPTION",
+                                            "VIEW_DARTS_INBOX",
+                                            "READ_TRANSCRIBED_DOCUMENT",
+                                            "LISTEN_TO_AUDIO_FOR_PLAYBACK",
+                                            "UPLOAD_JUDGES_NOTES",
+                                            "RETENTION_ADMINISTRATION"
+                                          ]
+                                        }
+                                      ],
+                                      "isActive": true
+                                    }""",
+                                mvcResult.getResponse().getContentAsString(),
+                                JSONCompareMode.NON_EXTENSIBLE
+        );
+    }
+
+    @Test
+    @WithMockSecurityContextWithEmailAddress(emailAddress = EMAIL_ADDRESS)
+    void getUserStateShouldFailWhenUserIsInactive() throws Exception {
```

Comment:
@hemantasharma1129 this is the current behaviour also. But is it correct? The `GET /userstate` response has an `isActive` field, but since the endpoint will always throw a 403 for inactive users, the `isActive` value can never be false, which makes the field essentially redundant.

Reply:
@chrisbellingham-hmcts I agree for /userstate the flag is redundant but the same service method is used in handlOauthCode method which expect the flag so don't think we have to make any changes for it.